routes.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  Image, StyleSheet, ScrollView, SafeAreaView, Platform,
+  Image,
+  StyleSheet,
+  ScrollView,
+  SafeAreaView,
+  Platform,
+  TouchableWithoutFeedback,
```

Comment:
@kostimarko `TouchableWithoutFeedback` is not used anywhere, can you please update this?